tests/getRaw.py

```diff
@@ -37,8 +38,10 @@ class GetRawTestCase(unittest.TestCase):
     """Testing butler raw image retrieval"""
 
     def setUp(self):
-        datadir = os.getenv("TESTDATA_DECAM_DIR")
-        assert datadir, "testdata_decam is not setup"
+        try:
+            datadir = lsst.utils.getPackageDir("TESTDATA_DECAM_DIR")
```

Comment:
The `_DIR` should not be included. Does it work like this? I think it should be something like
```python
datadir = lsst.utils.getPackageDir("testdata_decam")
```